### DISCUSSION TITLE
升级1.10.5 后提示需要升级 openResty，点击升级后面板失联

### DISCUSSION THREAD

**Discussion by yfdoor** [1Panel-dev/1Panel#5064] [Q&A]
### 1Panel 版本

1.10.5

### 问题描述

升级1.10.5 后可以进入面板，但是提示需要升级 openResty，点击升级后，面板失联。目前无法进入面板，

### 重现步骤

终端 1panel restore 回滚，问题依旧，无法进入面板。。

**Comment by yfdoor** [comment]
openResty 日志
`
l/openresty/nginx/conf/nginx.conf:32
nginx: [emerg] open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
2024/04/23 20:32:11 [emerg] 1#1: open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
nginx: [emerg] open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
2024/04/23 20:32:18 [emerg] 1#1: open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
nginx: [emerg] open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
2024/04/23 20:32:31 [emerg] 1#1: open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
nginx: [emerg] open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
2024/04/23 20:32:56 [emerg] 1#1: open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
nginx: [emerg] open() "/usr/local/openresty/1pwaf/data/conf/waf.conf" failed (2: No such file or directory) in /usr/local/openresty/nginx/conf/nginx.conf:32
`

**Comment by zhengkunwang223** [comment]
面板回滚当前无法回滚 OpenResty , OpenResty 需要你再升级一下
建议先不要用 OpenResty 代理面板，使用 IP：端口登录，等 OpenResty 升级成功之后再通过代理访问

你可以先手动执行 docker pull  1panel/openresty:1.21.4.3-2-1-focal
